### DISCUSSION TITLE
Local Port 1935 conflict with other ports used by other services.

### DISCUSSION THREAD

**Discussion by jnaizer62** [mrlt8/docker-wyze-bridge#1338] [General]
I'm trying to setup the Wyze Bridge in Docker and I'm getting the following error when I try to run. 
"Local Port 1935 conflict with other ports used by other services."
 What else would be using this port? I don't do any streaming other than PLEX.  Any ideas where I can check?

**Comment by mrlt8** [comment]
Do you have another instance of the bridge already running?

You can also skip port `1935` if you don’t use RTMP. You can disable it by comment it out or you can change the first value to use a different port:

```yaml
ports:
    - 1936:1935 # RTMP
    - 8554:8554 # RTSP
...
```

**Comment by Toei79** [comment]
same problem , but how you can edit that i mean on synology , i suppose its yml file where this its changed. 

![image](https://github.com/user-attachments/assets/58786a7f-63e5-4f93-9ade-aa6e5277d65f)

**Reply by dmkjr** [reply]
Change the local port to anything you want.  Container port can stay 1935.
